src/TheThingsNetwork.cpp

```diff
@@ -700,23 +919,25 @@ void TheThingsNetwork::showStatus()
 
 void TheThingsNetwork::configureEU868()
 {
-  sendMacSet(MAC_RX2, "3 869525000");
+  sendMacSet(MAC_RX2, 3, 869525000);
```

Comment:
I wonder if this was ever correct for OTAA. TTN V2 and V3 use SF12 for join-accept in RX2, and configure SF9 (for TTN frequency plans) through the join-accept.

In case of OTAA, the RX2 DR and which channels are enabled should not be left to default settings. Only in ABP this should be configured. I believe that `configureEU868()` is called also for OTAA, so that might be an issue. Right?

Reply:
Hi @johanstokking 
Yes, I agree with you. I didn't actually change that part, just added a private overload with one more parameter to save some program bytes here and there.

Actually, this line always confused me. The LoRa regional specs 1.0.2 define channel 9 (869.525) as default SF12. However, the same specification defines the downlink for Class B beacons @ 869.525 SF9 (DR3). I always thought it might have to do with the TTN server's default configuration (I used LorIoT so far)

Well, default join behavior defines confirmed messages on one of the three default frequencies (868.1, 868.3, and 868.5 MHz) and response within 5 or 6 seconds on RX1 or RX2 respectively. So if RX2 is wrong, it might work anyway. However, now that I added a method to change RX2, we should put that to default DR0 (Don't know the behavior of the modem if you give it nothing!). So a good point. 

So, should I make this change too? We can still configure everything like it's done now, but things like RX2 must stay default for OTAA to work as expected. Exceptional cases can use setRx2Channel.

Reply:
Thanks for your reply. Indeed I think we should use RX2 DR of 0 in case of OTAA. The server will configure the desired RX2 DR through the join-accept. For ABP, it needs to stay DR 3.

As for Class B beaconing vs RX2; those can be different indeed. I think that DR 3 was chosen because of a larger maximum payload.